plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceSerializerModern.java

```diff
@@ -526,7 +527,7 @@ public void parseDataSources(
                     }
                     dataSource = new DataSourceDescriptor(
                         registry,
-                        registry.getDefaultStorage(),
+                        configurationStorage,
```

Comment:
This change was intentional for the dbeaver/dbeaver-ee#1733. Please make sure this change doesn't break anything.

Reply:
We parse a data from `configurationStorage` here, not from some default storage, so, I'm sure that here should be `configurationStorage` which data we read.
The `getConfigurationStorages()` method is used both to get a collection of `configurationStorage`, which data we parse here (for each of them separately in this method), and in `registry.getDefaultStorage()` which was previously used with the only difference - `registry.getDefaultStorage()` chooses the default one from all configuration storages. In the same time `DatSourceDefaultStorage::isDefault` method always returns true, so all remote storages (if if there could be more than one for cloud) are default.

Of course, it's better to test both ce/ee and team edition.